work_queue/src/work_queue.c

```diff
@@ -278,6 +278,9 @@ struct work_queue_worker {
 struct work_queue_factory_info {
 	char *name;
 	int   connected_workers;
+	int   max_workers;
+	int   queried;
```

Comment:
`queried` gives me the idea that we talked directly to the factory to ask for something.
maybe `seen_at_catalog`?